lib/oneview-sdk/resource/api300/synergy/uplink_set.rb

```diff
@@ -14,7 +14,18 @@
 module OneviewSDK
   module API300
     module Synergy
+      # Uplink set resource implementation for API300 Synergy
       class UplinkSet < OneviewSDK::API200::UplinkSet
+
+        # Create a resource object, associate it with a client, and set its properties.
+        # @param [OneviewSDK::Client] client The client object for the OneView appliance
+        # @param [Hash] params The options for this resource (key-value pairs)
+        # @param [Integer] api_ver The api version to use when interracting with this resource.
+        def initialize(client, params = {}, api_ver = nil)
+          # Default values for api300
+          params[:type] ||= params['type'] ||= 'uplink-setV300'
```

Comment:
Instead of params, use @data['type'] ||= 'uplink-setV300'
Also, no need for using the symbol as well.